src/mlmc/pbs_process.py

```diff
@@ -0,0 +1,158 @@
+import os
+import sys
+import yaml
+import time
+import pickle
+import json
+from pathlib import Path
+
+
+src_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(src_path, '..', '..', 'src'))
+sys.path.append(os.path.join(src_path, '..', '..', 'test'))
+sys.path.append(os.path.join(src_path))
+
+
+class PbsProcess:
```

Comment:
Possibly we can view part of the proposed 'Job' class (see below) and the core of PbsProcess as a single object. Created in SamplingPoolPBS by adding the samples, then serialized, passed to qsub, and deserialized and executed in pbs_process.py. Parsing the arguments can be a separte function and we can have a class method as a factory. None initialization is boring.